data/autoyast_hanaperf/12sp5.xml.ep

```diff
@@ -61,7 +61,10 @@
   </networking>
   <partitioning config:type="list">
     <drive>
-      % if ($check_var->('MACHINE', 'ppc64le_legolas_2')) {
+      % if ($check_var->('MACHINE', 'ppc64le_blackcurrant_2')) {
+      <device>/dev/disk/by-id/wwn-0x5000c5007f9fae57</device>
```

Comment:
Could we think about a way to get disk id from a job setting instead of inferring it from `MACHINE` ? These settings sound specific to our specific test infrastructure.

Reply:
```
 <partitioning config:type="list">
    <drive>
      <device>
      % $get_required_var->('DISK_ID_1')
      </device>
```

Reply:
Good idea, I will fix and verify the solution after MU testing done.

Reply:
fixed
VR: http://10.67.129.4/tests/73109, http://10.67.129.4/tests/73108